pkg/porter/build_integration_test.go

```diff
@@ -110,7 +110,7 @@ func TestPorter_Build_ChecksManifestSchemaVersion(t *testing.T) {
 			defer p.Close()
 
 			// Make a bundle with the specified schemaVersion
-			p.TestConfig.TestContext.AddTestDirectoryFromRoot("tests/testdata/mybuns", "/")
+			p.TestConfig.TestContext.AddTestDirectoryFromRoot("tests/testdata/mydb", "/")
```

Comment:
Can you help me understand why this was changed to user a different bundle? If the test worked before with mybuns, I would expect it to still work with mybuns with a referenced image. I am concerned that changing the test bundle may be hiding a regression or problem in Porter.

Reply:
I changed it back